src/styles/styles.js

```diff
@@ -26,13 +26,11 @@ const picker = {
     fontFamily: fontFamily.EXP_NEUE,
     fontSize: variables.fontSizeNormal,
     lineHeight: variables.fontSizeNormalHeight,
-    paddingHorizontal: 11,
     paddingBottom: 8,
     paddingTop: 23,
+    paddingHorizontal: 0,
     height: variables.inputHeight,
     borderWidth: 0,
-    borderRadius: variables.componentBorderRadiusNormal,
-    textAlign: 'left',
```

Comment:
This is necessary. Please revert it back. AFAIK, we need to use `textAlign:left` to force the LTR mode. I don't remember the details but I reviewed the LTR mode PR so I know.